tests/babel/fixtures/dynamicAttr.js

```diff
@@ -11,9 +11,9 @@ html`
   <div id="str" ${className}="blub" />
 `
 
-var x = 'disabled'
+var x = { disabled: 'disabled' }
 html`
   <button ${x} id="lol" />
```

Comment:
I'm not sure if this is supposed to be allowed as it creates ambiguity with props spread 
```js
var obj = { prop: 'value' }
var tree = html`<button ${obj} />`
```
There is also some inconsistency within the previous implementations:
```js
var x = 'disabled'
var tx = html`<button ${x}>foobar</button>`
// tx.outerHTML == <button disabled="disabled">foobar</button>

var y = false
var ty = html`<button disabled=${x}>foobar</button>`
// ty.outerHTML == <button>foobar</button>

tx.outerHTML !== ty.outerHTML
```
Regardless, I can spend some time making prop spread and this shorthand notation to work together

Reply:
It is supposed to be allowed, yes